vvsvintsitsky/shop-react-redux-cloudfront
# Homework-2: Intergrate with serverless API for products

### Link to the [web app](https://d210b17oei28zu.cloudfront.net/)

### In order to test integration please do the following
* open web app page
* open network tab in devtools
* see a request for **/products** endpoint that returns a collection of products



To test automated build & deploy open serverless.yml file and change service name and bucket name in lines 1 and 17 respectively, make sure that names are unique across all buckets in S3, then use **cloudfront:update:build:deploy:nc** script.

Please note that YARN is used as a package manager.
